### PR TITLE
Make the root namespaces consistent

### PR DESCRIPTION
**Pull request by AndriySvyryd**

### CODE REVIEW

File: test/EntityFramework.Microbenchmarks.EF6/EntityFramework.Microbenchmarks.EF6.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>EntityFramework.Microbenchmarks.EF6</RootNamespace>
-    <AssemblyName>EntityFramework.Microbenchmarks.EF6</AssemblyName>
+    <AssemblyName>EntityFramework.Microbenchmarks</AssemblyName>
```
Comment (smitpatel) [inline test/EntityFramework.Microbenchmarks.EF6/EntityFramework.Microbenchmarks.EF6.csproj]: csproj & xproj file has mismatch for AssemblyName and RootNamespace.

Comment (AndriySvyryd) [inline test/EntityFramework.Microbenchmarks.EF6/EntityFramework.Microbenchmarks.EF6.csproj]: Yes, it should be the other way around

File: src/EntityFramework.Relational/EntityFramework.Relational.xproj [hunk 2]
```diff
@@ -10,6 +10,12 @@
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
     <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
   </PropertyGroup>
+  <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <AssemblyName>EntityFramework.Relational</AssemblyName>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <RootNamespace>Microsoft.Data.Entity.Relational</RootNamespace>
```
Comment (ajcvickers) [inline src/EntityFramework.Relational/EntityFramework.Relational.xproj]: But we decided to not have this namespace? ;-)

**Comment by ajcvickers** [comment]
:shipit: for now. We can revisit where needed when the namespace structure emerges from the wash.

File: src/EntityFramework.Relational/EntityFramework.Relational.xproj (hunk 2 above)
Comment (AndriySvyryd) [inline src/EntityFramework.Relational/EntityFramework.Relational.xproj]: So this will be one more place to be changed

**Pull request merged**